### DISCUSSION TITLE
Failed to compile MacOS 11.2

### DISCUSSION THREAD

**Discussion by gastonet** [AlexandreRouma/SDRPlusPlus#64] [Q&A]
Hi everyone, I'm trying to do my best but I can't solve this error when compiling:

`ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [core/libsdrpp_core.dylib] Error 1
make[1]: *** [core/CMakeFiles/sdrpp_core.dir/all] Error 2
make: *** [all] Error 2
gastonet@MBP-de-GE SDRPlusPlus %`

I think is related to the architecture but I'm not a guru on this.

Any help will be much appreciated.

**Comment by colbyAtCRI** [comment]
Well, I got it to compile and it ran once (without a radio). Key items in cmake seem to be 

CMAKE_CXX_COMPILER /usr/local/bin/g++-10

CMAKE_C_COMPILER /usr/local/bin/gcc-10

clang build is boned, /usr/bin/c++ is clang in disguise. You get further but it still fails.

I had trouble with core/src/signal_path/sink.h Inside it references
#include <event.h>
which happily grabbed an event.h from some random directory. I changed it to this
#include "../event.h"
which points to the intended file.

**Reply by AlexandreRouma** [reply]
I should probably put most of that stuff in other dirs
didn't think that name would cause issue, but guess OSX really is being annoying :/
